mender/templates/api-gateway-configmap.yaml

```diff
@@ -473,6 +495,10 @@ data:
         compression:
           compress: true
 
+        ratelimit:
+          ratelimit:
```

Comment:
I believe this is case sensitive:
```suggestion
          rateLimit:
```

Reply:
Fixed, it was a copy and paste from `saas`.
https://github.com/mendersoftware/saas/pull/672